resteasy/pom.xml

```diff
@@ -98,6 +98,11 @@
       <artifactId>beadledom-swagger1</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.cerner.beadledom</groupId>
+      <artifactId>beadledom-guice-dynamicbindings</artifactId>
```

Comment:
nit. This dependency should be above beadledom-swagger1.

Reply:
It can be removed completely after some of the changes I suggested in the tests.

Reply:
https://github.com/cerner/beadledom/pull/143/commits/75d364f068be5b48f12a864aa2a8bd938fb929b8